src/popup/components/Header.vue

```diff
@@ -85,60 +91,46 @@ export default {
     BtnPlain,
     BtnIcon,
   },
-  subscriptions() {
-    return {
-      superheroNotifications: this.$store.state.observables.notifications,
-    };
-  },
-  computed: {
-    ...mapGetters(['isLoggedIn']),
-    ...mapState(['notifications', 'pageTitle']),
-    truncateStr() {
-      const { pageTitle, title } = this;
-      return pageTitle || (title ? this.$t(`pages.titles.${title}`) : '');
-    },
-    title() {
-      return this.$route.meta.title;
-    },
-    isModalHeader() {
-      return this.$route.meta.isModalHeader;
-    },
-    showBack() {
-      return (
-        (this.$route.meta.backButton !== undefined ? this.$route.meta.backButton : true)
-        && this.title
-      );
-    },
-    showMore() {
-      return this.$route.name !== 'more' && !this.$route.meta?.closeButton;
-    },
-    showNotifications() {
-      return this.$route.name !== 'notifications' && !this.$route.meta?.hideNotificationsIcon;
-    },
-    isLogoDisabled() {
-      return this.$route.name === 'account';
-    },
-  },
-  methods: {
-    back() {
-      if (this.$store.state.route.from.path === '/') {
-        this.$router.push(this.isLoggedIn ? '/account' : '/');
-        return;
+  setup(props, { root }) {
+    const isLoggedIn = useGetter('isLoggedIn');
+    const routeMeta = computed(() => root.$route.meta as WalletRouteMeta);
+    const showHeaderNavigation = computed(() => !!routeMeta.value?.showHeaderNavigation);
+    const isLogoDisabled = computed(() => root.$route.name === ROUTE_ACCOUNT);
+    const titleTruncated = computed(
+      () => routeMeta.value?.title ? root.$t(`pages.titles.${routeMeta.value.title}`) : '',
+    );
+
+    function back() {
+      if (root.$store.state.route.from.name === ROUTE_INDEX) {
+        return root.$router.push({
+          name: (isLoggedIn.value) ? ROUTE_ACCOUNT : ROUTE_INDEX,
```

Comment:
```suggestion
          name: isLoggedIn.value ? ROUTE_ACCOUNT : ROUTE_INDEX,
```

Reply:
Actually some wise developer once said to me that if I wrap the test in ternary operation in braces it will be easier and faster to analyse the code we see. After years of coding I'm sure that's true.

Because you follow the code from top to bottom, from left to right. So when you see something like this:
```
const a = something...
```
you assume that this is a regular assignment. But When your eyes rich something like this:
```
const a = (something...
```
you immediately know that there will be some logic without seeing whole line.